app/src/main/java/com/mealsmadeeasy/ui/home/discover/SearchActivity.kt

```diff
@@ -86,4 +89,9 @@ class SearchActivity : BaseActivity() {
         onBackPressed()
         return true
     }
+
+    override fun onSaveInstanceState(outState: Bundle) {
+        super.onSaveInstanceState(outState)
+        outState.putString(KEY_SAVED_QUERY,querySubject.value)
```

Comment:
So this will save the last search query, but we're still not handling configuration changes because we don't restore this value. So in one of the onCreate methods, you need to use the `savedInstanceState` bundle to restore this field into the query of the search view.

Also, nit: can you add a space after the comma here?